src/rules/prettier.ts

```diff
@@ -36,12 +36,23 @@ export default {
     return {
       Program() {
         const sourceCode = context.sourceCode.text
-        const formatted = format(sourceCode, {
-          filepath: context.filename,
-          ...(context.options[0] || {}),
-        })
+        try {
+          const formatted = format(sourceCode, {
+            filepath: context.filename,
+            ...(context.options[0] || {}),
+          })
 
-        reportDifferences(context, sourceCode, formatted)
+          reportDifferences(context, sourceCode, formatted)
+        }
+        catch (error) {
+          context.report({
+            loc: {
+              start: { line: 1, column: 0 },
```

Comment:
Is it possible to get the position information from Prettier or the error project and report it to the place? And then we could fallback to the start of the file if failed

Reply:
The error message looks like this:

```
CssSyntaxError: Unknown word (3:3)
  1 | .foo {
  2 |   color: green;
> 3 |   ...
    |   ^
  4 | }
  5 |
```

I've updated it to get a more accurate location.